eq-author/src/components/Forms/Input/index.js

```diff
@@ -29,6 +29,32 @@ const radioButton = css`
   ${multipleChoiceOption};
   border-radius: 100%;
 
+  ${({ variant }) =>
+    variant === "radioBox" &&
+    `
+    width: 1.25em;
+    height: 1.25em;
+    border-radius: 100%;
+    outline: none;
+    position: absolute;
+    overflow: hidden;
+    padding: 0;
+    display: flex;
+
+    :before {
+      content: "";
+      display: block;
+      width: 80%;
+      height: 80%;
+      margin: 10% auto;
+      border-radius: 100%;
+      background: #dcdcdc;
+    }
+    :checked:before {
+      background: #323232;
+    }
```

Comment:
Hi Sam. Sorry, I should have noticed this last time I had a look. Inside this file, src/constants/theme.js there's an object called colors. Are there any colours in there that you could use instead of this new one?

You can speak to Joe if you're not sure -- iirc we had a conversation at a retrospective (before you joined the team I think - so you wouldn't know about this) about trying to reuse as many colours as we can to avoid adding new ones for single uses.

If it turns out none of them are appropriate, you can add your new colour into that object and reference it in the code that you're written. The benefit is that we can centralise our colours so that if we ever need to bulk change them we can do in a single place.